src/selectMenus/sampleSelectMenu.ts

```diff
@@ -0,0 +1,14 @@
+import { SelectMenuHandler } from "../handler";
+import { SelectMenuInteraction } from "discord.js";
+
+class ExampleSelectMenu extends SelectMenuHandler {
+	constructor() {
+		super("exampleSelectMenu");
+	}
+
+	async execute(interaction: SelectMenuInteraction, args: string[]): Promise<void> {
+		await interaction.reply(`You selected the following options: ${args}`);
+	}
```

Comment:
This is not correct. The arg is actually something the command left for itself (useful for if a command has multiple select menus it uses). 

Instead this should look more like:
```suggestion
	async execute(interaction: SelectMenuInteraction, args: string[]): Promise<void> {
		await interaction.reply(`You selected the following options: ${interaction.values}`);
	}
```